app/models/genre.rb

```diff
@@ -0,0 +1,12 @@
+class Genre < ApplicationRecord
```

Comment:
At the moment we can create Genres with the same name. Does that seem reasonable to you?

Reply:
Nope! Adding the validation in the next commit.